src/app/clusters/resource-monitoring-server/resource-monitoring-server.cpp

```diff
@@ -300,7 +300,7 @@ CHIP_ERROR Instance::Read(const ConcreteReadAttributePath & aPath, AttributeValu
         ReturnErrorOnFailure(aEncoder.Encode(mLastChangedTime));
         break;
     }
-    case Attributes::ReplacementProductList::Id: {
+    case HepaFilterMonitoring::Attributes::ReplacementProductList::TypeInfo::GetAttributeId(): {
```

Comment:
This is worth a comment explaining why the HepaFilterMonitoring bit is being used.  Or you could leave the definition of `Attributes::ReplacementProductList::Id` around.....

Reply:
Decided to leave the definition around, makes the code easier to read since all the other attributes are listed there as well